hooks/post-command

```diff
@@ -11,13 +11,30 @@ if [[ "${#CORE_DUMPS[@]}" > 0 ]]; then
     echo "--- List of core dumps"
     ls -lah "${CORE_DUMPS[@]}"
 
-    # Start compressing/collecting corefiles in the background
+    # Start a subshell for compressing/collecting corefiles in the background in parallel
+    (
     PIDS=()
     for COREFILE in "${CORE_DUMPS[@]}"; do
         compress_bundle "${COREFILE}" &
```

Comment:
Would it be better to do each upload immediately after each compression, or was there a reason these are separate phases?
```suggestion
        (
        compress_bundle "${COREFILE}"
        buildkite-agent artifact upload "$(compressed_bundle_names ${COREFILE})"
        ) &
```

Reply:
I think this should be fine